app/controllers/admin/tags_controller.rb

```diff
@@ -57,12 +64,11 @@ def badges_for_options
     end
 
     def tag_params
-      allowed_params = %i[
-        id supported rules_markdown short_summary pretty_name bg_color_hex
-        text_color_hex user_id alias_for badge_id requires_approval
-        category social_preview_template wiki_body_markdown submission_template
-      ]
-      params.require(:tag).permit(allowed_params)
+      params.require(:tag).permit(ALLOWED_PARAMS)
+    end
+
+    def tag_alias_updated?
+      tag_params[:alias_for].present?
```

Comment:
This seems sufficient because if we remove an alias we can't undo the previous re-tagging as we don't know anymore which articles had the original tags and which had the alias before.